component/core/src/main/java/org/exoplatform/social/core/attachment/AttachmentServiceImpl.java

```diff
@@ -400,9 +410,13 @@ private void moveAttachments(String sourceObjectType,
       attachments.forEach(attachment -> {
         String altText = attachment.getAltText();
         String format = attachment.getFormat();
+        String linkUrl = attachment.getLinkUrl();
+        String linkTarget = attachment.getLinkTarget();
         Map<String, String> properties = new HashMap<>();
         properties.put(ATTACHMENT_ALT_TEXT, altText);
         properties.put(ATTACHMENT_FORMAT, format);
+        properties.put(ATTACHMENT_LINK_URL, linkUrl);
+        properties.put(ATTACHMENT_LINK_TARGET, linkTarget);
```

Comment:
Add `StringUtils.isNotBlank` test before adding to properties Map please to improve the used storage space. (for those 4 properties)